IGC/DebugInfo/StreamEmitter.cpp

```diff
@@ -322,7 +322,11 @@ class VISAAsmBackend : public MCAsmBackend {
   }
 #endif
 
+#if LLVM_VERSION_MAJOR >= 14
+  bool writeNopData(raw_ostream &OS, uint64_t Count, const MCSubtargetInfo *STI) const override {
+#else
   bool writeNopData(raw_ostream &OS, uint64_t Count) const override {
+#endif
```

Comment:
Easy wrapper for this, just proxy `writeNopData` call like this:

```cpp
    T f(a, b)
    {
#if LLVM14
        return g(a, b);
#else
        return g(a);
#endif
    }
```

Reply:
> Easy wrapper for this, just proxy `writeNopData` call like this:

Could you elaborate on how to do this when the function is `override`'ing a virtual function? I can't see how I can use a wrapper for this.